allauth/socialaccount/providers/untappd/client.py

```diff
@@ -26,11 +29,16 @@ def get_access_token(self, code):
         if self.access_token_method == 'GET':
             params = data
             data = None
+        # Allow custom User Agent to comply with Untappd API
+        settings = app_settings.PROVIDERS.get(UntappdProvider.id, {})
+        headers = {
+            'User-Agent': settings.get('USER_AGENT', 'django-allauth-header')}
```

Comment:
Shouldn't the default agent be just "django-allauth" instead of "django-allauth-header" ?

Reply:
Thanks for the feedback, seems that the default agent 'django-allauth' is indeed working just fine. Changed it in commit 9a22b4d